exercises/hello-world/HINTS.md

```diff
@@ -30,23 +29,23 @@ The test fails, which makes sense since you've not written any code yet.
 
 The failure looks like this:
 
-    1) Hello World says hello world with no name
+    1) Hello World says hello world 
        Message:
           Expected undefined to equal 'Hello, World!'.
 
 There's more, but this is the most important part.
 
 Take a look at that first line:
 
-    1) Hello World says hello world with no name
+    1) Hello World says hello world 
 
 Now look at the test definition again:
 
-    it('says hello world with no name', function() {
+    it('says hello world', function() {
       // ... more code here ...
     });
 
-The text 'says hello world with no name' is repeated.
+The text 'says hello world' is repeated.
 This is how you know which test failed.
```

Comment:
Again, since there's only one test now, this should be changed to 'This is how you know the test failed.'